### ISSUE TITLE
react stateless functional component snippet

### ISSUE THREAD

**Issue by audiolion** [dsznajder/vscode-es7-javascript-react-snippets#17]
Would you be adverse to adding the non fat arrow syntax version of a stateless functional component?

```
function $1(props) {
  return (
    <div>$2</div>
  );
}

export default $1
```

I ask because [flow recommends this style](https://flow.org/en/docs/react/components/#toc-stateless-functional-components)

**Comment by dsznajder** [comment]
Hi @audiolion,  I'm not a very big fan of writing `function` word nowadays in JS. We have Babel, and other stages to transpile arrow functions. However, if we figure out how to name those you can create a PR :) I thought about `pfc` -> pureFunctionComponent and so on. What do you think?

**Comment by audiolion** [comment]
Yeah @dsznajder I don't too much either, but it is a valid way and if some codebases have opinions about styling functions then perhaps this would be useful. I suppose we could wait to see if some other people would vote 👍 on whether they would want this and if adding another keyword is worth it
